doc/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Can we bundle this config with aegir and load the config from there?

It would be better than copy/pasting this file into every repo.

Reply:
Good point, once https://github.com/ipfs/aegir/pull/1255 is merged then we can upgrade `aegir` here to `39.0.6` and then merge.